### PR TITLE
Implement java.util.logging bridge

### PR DESCRIPTION
**Pull request by andrzejressel**
Useful for `zio-telemetry` - Java OpenTelemetry library is using JUL (https://togithub.com/open-telemetry/opentelemetry-java/issues/5715)

### CODE REVIEW

File: docs/jul-bridge.md
Comment (justcoon) [inline docs/jul-bridge.md:1]: page needs to be added to
https://github.com/zio/zio-logging/blob/master/docs/sidebars.js

File: jul-bridge/src/main/scala/zio/logging/jul/bridge/ZioLoggerRuntime.scala [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+object ZioLoggerRuntime {
+
+  private val logLevelMapping: Map[Level, LogLevel] = Map(
+    Level.SEVERE  -> LogLevel.Fatal,
+    Level.WARNING -> LogLevel.Warning,
+    Level.INFO    -> LogLevel.Info,
+    Level.CONFIG  -> LogLevel.Info,
+    Level.FINE    -> LogLevel.Debug,
+    Level.FINER   -> LogLevel.Debug,
+    Level.FINEST  -> LogLevel.Debug,
```
Comment (justcoon) [inline jul-bridge/src/main/scala/zio/logging/jul/bridge/ZioLoggerRuntime.scala:99]: should not we map this one to `LogLevel.Trace` ?

File: docs/jul-bridge.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+<br/>
+
+
+JUL bridge with custom logger can be setup:
+
+```scala
+import zio.logging.jul.bridge.JULBridge
+import zio.logging.consoleJsonLogger
+
+val logger = Runtime.removeDefaultLoggers >>> consoleJsonLogger() >+> Slf4jBridge.init()
```
Comment (justcoon) [inline docs/jul-bridge.md:61]: Slf4jBridge.init() -> JULBridge.init()

File: README.md [hunk 3]
```diff
@@ -76,6 +76,12 @@ Other modules:
     When to use this module: you are already using Java Platform/System Logger in some other project, and you like to have same log outputs.
     See [Java Platform/System Logger](docs/jpl.md) section for more details.
 
+* java.util.logging bridge - with this logging bridge, it is possible to use `zio-logging` for JUL loggers (usually third-party non-ZIO libraries), add the one of following lines to your `build.sbt` file:
+
+    ```scala
+    // JUL bridge
+    libraryDependencies += "dev.zio" %% "zio-logging-jul-bridge" % "2.2.2"
+    ```
```
Comment (justcoon) [inline README.md:84]: is good to add something like

```
See [java.util.logging bridge](docs/jul-bridge.md) section for more details.
```

for quick navigation

**Comment by justcoon** [comment]
hi @andrzejressel, thank you very much  for contribution

File: jul-bridge/src/main/scala/zio/logging/jul/bridge/ZioLoggerRuntime.scala (hunk 1 above)
Comment (andrzejressel) [reply jul-bridge/src/main/scala/zio/logging/jul/bridge/ZioLoggerRuntime.scala:99]: I'll try to decipher values from SLF4J's bridge and use them here.

Comment (justcoon) [reply jul-bridge/src/main/scala/zio/logging/jul/bridge/ZioLoggerRuntime.scala:99]: based on what is see here https://github.com/qos-ch/slf4j/blob/master/jul-to-slf4j/src/main/java/org/slf4j/bridge/SLF4JBridgeHandler.java#L114

it look like JUL FINEST is logged as slf4j Trace

so not sure, or do you mean something else?

Comment (andrzejressel) [reply jul-bridge/src/main/scala/zio/logging/jul/bridge/ZioLoggerRuntime.scala:99]: Yep, they also map `SEVERE` to `ERROR`, but they don't have `FATAL` equivalent - so `FINEST` will be the only change.

File: README.md (hunk 3 above)
Comment (andrzejressel) [reply README.md:85]: Fixed

File: docs/jul-bridge.md (hunk 2 above)
Comment (andrzejressel) [reply docs/jul-bridge.md]: Fixed

File: jul-bridge/src/main/scala/zio/logging/jul/bridge/ZioLoggerRuntime.scala (hunk 1 above)
Comment (andrzejressel) [reply jul-bridge/src/main/scala/zio/logging/jul/bridge/ZioLoggerRuntime.scala]: Fixed

File: docs/jul-bridge.md
Comment (andrzejressel) [reply docs/jul-bridge.md:1]: Fixed

**Review by justcoon — APPROVED** [review]

**Comment by justcoon** [comment]
hi @andrzejressel, is PR ready for merge, or do you want to add something more?

**Comment by andrzejressel** [comment]
I think it's ready

**Pull request merged**